package.json

```diff
@@ -39,10 +40,13 @@
   },
   "files": [
     "devtools",
+    "debug",
```

Comment:
Thinking this might cause issues since there is both `debug.js` and `debug/index.js`? Not sure.